common/models/templates/package.json

```diff
@@ -46,7 +46,8 @@
     "build": "gosh ./build.sh",
     "tsc": "tsc",
     "mocha": "mocha",
-    "test": "mocha -r test/helpers.js"
+    "test": "mocha -r test/helpers.js",
+    "c8": "c8"
```

Comment:
Can we make these use build.sh, and then we don't need `tsc`, `mocha`, or `c8` scripts also:

```
"test": "gosh ./build.sh test"
```

Reply:
I didn't even notice the `test` bit.

That said, I like having `tsc`, `mocha` and the like so I can directly run those calls on occasion outside of build.sh scripts.  I definitely use this from time to time with `mocha` (and `karma` for DOM-reliant Web components).

Reply:
You can of course do `$KEYMAN_ROOT/node_modules/.bin/tsc` etc (or even add your `node_modules/.bin` to your path)